src/status_im/ui/screens/add_new/new_public_chat/subs.cljs

```diff
@@ -0,0 +1,12 @@
+(ns status-im.ui.screens.add-new.new-public-chat.subs
+  (:require [re-frame.core :as re-frame]
+            [status-im.ui.screens.add-new.new-public-chat.db :as db]
+            [cljs.spec.alpha :as spec]))
+
+(re-frame/reg-sub
+  :new-topic-error-message
```

Comment:
this is an interesting use of a subscription, to check a schema against the actual data :thinking: cool stuff!

Reply:
@rcullito thanks for the pointer to `show-popup` in `utils.utils`! Unfortunately this is an alert-based popup, and we'd want to have multi-item clickable popups, more like proper context menus.